psscriptanalyzer/Invoke-HealthCheck.ps1

```
@@ -0,0 +1,62 @@
+param(
+    [string]$Target = "demoapp-web",
+    [int]$Retries = 3,
```

PSScriptAnalyzer (1.25.0)

[warning] The parameter 'Retries' has been declared but not used.

(PSReviewUnusedParameter)